python/lsst/summit/utils/nightReport.py

```diff
@@ -107,9 +113,20 @@ def _load(self, filename):
         """
         with open(filename, "rb") as f:
             loaded = pickle.load(f)
-        self.data, self._expRecordsLoaded, self._obsInfosLoaded, dayObs = loaded
+        self.data = loaded['data']
+        self._expRecordsLoaded = loaded['_expRecordsLoaded']
+        self._obsInfosLoaded = loaded['_obsInfosLoaded']
+        dayObs = loaded['dayObs']
+        loadedVersion = loaded.get('version', 0)
+
         if dayObs != self.dayObs:
             raise RuntimeError(f"Loaded data is for {dayObs} but current dayObs is {self.dayObs}")
+        if loadedVersion < self._version:
+            self.log.critical(f"Loaded version is {loadedVersion} but current version is {self._version}."
+                              " Check carefully for compatibility issues/regenerate your saved report!")
```

Comment:
Given that you're using pickle, I think it's highly likely you'll get errors when reading incompatible versions long before you get here, since that's quite easy to get when the pickled data structures change.

Reply:
As discussed OOB, I'm being careful to only pickle the scraped data and no code at all, so hopefully this will still work OK, but the warning seemed like a nice feature to add anyway, just in case.